packages/core/src/NodeExecuteFunctions.ts

```diff
@@ -388,14 +388,14 @@ async function proxyRequestToAxios(
 			.then((response) => {
 				if (configObject.resolveWithFullResponse === true) {
 					resolve({
-						body: response.data,
+						body: response.data === '' ? undefined : response.data,
```

Comment:
what if the service is returning an empty response?

Reply:
We are keeping compatibility with what the request library did. Empty responses are set to undefined. n8n works fine with this behavior.

Reply:
Ah, as a side note, this only happens if the response is set to `json`.

If you set the response type to `String` and it's empty, no changes will happen and you will receive an empty string as output.

Reply:
something to look out for.. but request seems to behave differently with empty bodies depending on encoding 
https://github.com/request/request/blob/3c0cddc7c8eb60b470e9519da85896ed7ee0081e/tests/test-emptyBody.js#L19
https://github.com/request/request/blob/3c0cddc7c8eb60b470e9519da85896ed7ee0081e/request.js#L1151
maybe double check this since we are setting encoding for binary requests

Reply:
Thanks for pointing that out @mutdmour 

This could break if expected output is a binary file with no length. I just pushed an update that covers this.